.no-header.php-cs-fixer.dist.php

```diff
@@ -11,29 +11,29 @@
  * the LICENSE file that was distributed with this source code.
  */
 
+use CodeIgniter\CodingStandard\CodeIgniter4;
 use Nexus\CsConfig\Factory;
+use Nexus\CsConfig\Fixer\Comment\SpaceAfterCommentStartFixer;
+use Nexus\CsConfig\FixerGenerator;
 use PhpCsFixer\Finder;
-use Utils\PhpCsFixer\CodeIgniter4;
-use Utils\PhpCsFixer\Fixer\Comment\SpaceAfterCommentStartFixer;
 
 $finder = Finder::create()
     ->files()
     ->in([
         __DIR__ . '/admin',
         __DIR__ . '/app',
         __DIR__ . '/public',
-    ]);
+    ])
+    ->notName('#Logger.php$#');
```

Comment:
is `.` need to be escaped? 

```suggestion
    ->notName('#Logger\.php$#');
```